cmd/wrapcontract/usage.md

```diff
@@ -0,0 +1,18 @@
+This command takes the bytecode that we want to be deployed for a new contract, and wraps its into the bytecode that we can use to create the contract.
```

Comment:
I find that using the word `bytecode` is a bit vague, what do you think of "runtime/deployed" bytecode and "creation bytecode"?

```suggestion
This command takes the runtime bytecode, the bytecode deployed on-chain, as input and converts it into creation bytecode, the bytecode used to create the contract
```